src/lib/db/feature-toggle-client-store.ts

```diff
@@ -57,6 +57,7 @@ export default class FeatureToggleClientStore
         featureQuery?: IFeatureToggleQuery,
         archived: boolean = false,
         isAdmin: boolean = true,
+        includeStrategyIds?: boolean,
```

Comment:
This and the other changes in this file are here to facilitate getting strategy ids in addition to the strategy definition. (see the pr description for more info)